Gemfile.lock

```diff
@@ -73,16 +74,32 @@ GEM
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     parallel (1.19.1)
+    parser (2.7.1.3)
+      ast (~> 2.4.0)
```

Comment:
This doesn't seem to be a dependency for `rubocop`, could you please elaborate?

Reply:
I suspect that because `rubocop` requires `rubocop-ast` this is something coming from that.  This is hinted at in the gemspec file for Rubocop.